src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx

```diff
@@ -58,9 +58,14 @@ export const Unstake = ({ assetId, apr, accountSpecifier, validatorAddress }: Un
   const asset = useAppSelector(state => selectAssetByCAIP19(state, assetId))
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
   const totalBondings = useAppSelector(state =>
```

Comment:
nit : maybe change variable named `totalBondings` to something more related to the used selector

Reply:
Good idea! Changed it to `cryptoStakeBalance` to match `cryptoStakeBalanceHuman`